src/routes/Dashboard/index.tsx

```diff
@@ -43,58 +43,44 @@ function Dashboard ({ match }: DashboardProps) {
   })
   const { projects, libraries, recentLibraries } = data
   if (loading) return <Loading />
-  const renderWidgets = React.useCallback(
```

Comment:
Usage of hooks after branching logic is not allowed by React. `useCallback` hook is being used solely for performance reasons, so the easiest way to solve the problem is to drop the hook usage at all. We can and should care for performance improvements in the application, though, but that responsibility should probably be placed elsewhere - such as splitting the code in minor and pure components.